Ato-visionOS/Ato-visionOS/Source/View/SplitPeriodicView.swift

```diff
@@ -15,58 +15,101 @@ import SwiftUI
 struct SplitPeriodicView: View {
     
     // MARK: - State
-    /// 사용자가 선택한 원소
-    @State private var selectedElement: Element? = nil
+    // 사용자가 선택한 원소
+    @State private var selectedAtom: DetailAtomModel? = nil
 
     // MARK: - Body
     var body: some View {
-        GeometryReader { geometry in
+        GeometryReader {geometry in
+            let size = geometry.size
+            let width: CGFloat = geometry.size.width
+            let height: CGFloat = geometry.size.height
+//            let dynamicCorner = min(30, min(width, height) * 0.05)
             
-            // MARK: - 레이아웃 계산
-            
-            let totalWidth = geometry.size.width
-            let leftWidth = totalWidth * 0.7           // 좌측 주기율표 영역
-            let rightWidth = totalWidth - leftWidth    // 우측 상세보기 영역
-            
-            HStack(spacing: 0) {
-                
+            HStack(spacing: 10) {
                 // MARK: - 주기율표 뷰 (왼쪽)
+                
                 PeriodicTableView(
-                    selectedElement: $selectedElement,
+                    selectedAtom: $selectedAtom,
                     elementsGrid: elementsGrid,
-                    containerWidth: leftWidth,
-                    containerHeight: geometry.size.height
+                    width: width * 0.71,
+                    height: height
                 )
-                .frame(minWidth: leftWidth)
-
-                Divider() // 가운데 구분선
+                .frame(width: width * 0.71, height: height)
+                .background(
+                  LinearGradient(
+                    stops: [
+                      Gradient.Stop(color: .white.opacity(0.37), location: 0.00),
+                      Gradient.Stop(color: Color(red: 0.45, green: 0.45, blue: 0.45).opacity(0.42), location: 0.95),
+                    ],
+                    startPoint: UnitPoint(x: 0, y: -0.06),
+                    endPoint: UnitPoint(x: 1, y: 1)
+                  )
+                )
+                .cornerRadius(55)
+                
                 
+                Color.clear
+                    .frame(width: width * 0.001)
```

Comment:
가운데 빈 공간을 만들고 싶은거라면 
```suggestion
                spacer()
                    .frame(width: width * 0.001)
```
spacer를 쓰는 걸 더 추천드려요!
spacer 화면에 렌더링할 실제 콘텐츠가 없고, 단순히 공간만 차지하도록 설계되어 있어서
즉, 메모리/CPU 사용량이 거의 없고, 단순히 배치만을 위한 뷰로 사용됩니다!

Color.clear는 View로 처리되기때문에 메모리측면에서 spacer가 더 유리합니다!